tests/Microsoft.Bot.Builder.AI.LUIS.Tests/LuisRecognizerTests.cs

```diff
@@ -962,7 +947,19 @@ public async Task Telemetry_ConvertParms()
             Assert.AreEqual(((Dictionary<string, double>)telemetryClient.Invocations[0].Arguments[2])["luis"], 1.0001);
         }
 
-
+        private static TurnContext GetContext(string utterance)
+        {
+            var b = new TestAdapter();
```

Comment:
Can we rename this variable from `b` to `testAdapter` (or something similar)?

Reply:
ACK

Reply:
Done